src/components/AvatarStack/__stories__/AvatarStack.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import {faker} from '@faker-js/faker/locale/en';
+import type {Meta, StoryFn} from '@storybook/react';
+
+import type {AvatarProps, AvatarSize} from '../../Avatar';
+import {AVATAR_SIZES, Avatar, DEFAULT_AVATAR_SIZE} from '../../Avatar';
+import {Tooltip} from '../../Tooltip';
+import {AvatarStack} from '../AvatarStack';
+
+type ComponentType = React.ComponentProps<typeof AvatarStack> & {avatarSize: AvatarSize};
+
+function getChildren({
+    count = faker.number.int({min: 1, max: 30}),
+    avatarSize = DEFAULT_AVATAR_SIZE,
+}: Partial<{count: number; avatarSize: AvatarSize}>) {
+    return faker.helpers.uniqueArray(
+        () => (
+            <Avatar
+                imgUrl={faker.image.avatar()}
+                size={avatarSize}
+                borderColor={'var(--g-color-line-generic-solid)'}
+                aria-label={'For tests'}
+                alt={'For tests'}
+            />
+        ),
+        count,
+    );
+}
+
+export default {
+    title: 'Components/Data Display/AvatarStack',
+    component: AvatarStack,
+    args: {
+        overlapSize: 's',
+        avatarSize: DEFAULT_AVATAR_SIZE,
+    },
+    argTypes: {
+        avatarSize: {
+            control: 'select',
+            options: Object.keys(AVATAR_SIZES),
+            name: 'Size of avatar',
+            description: 'Not part of component API',
+        },
+    },
+} as Meta<ComponentType>;
```

Comment:
Is it necessary to create this type? `typeof AvatarStack` wouldn't work?

Reply:
Just don't want to duplicate same code in all other places it is used